common/output-model.cpp

```diff
@@ -82,32 +94,39 @@ void output_model::thread_loop()
                                 }
                             }
 
-                            if (!parsed_ok)
+                            if( ! parsed_ok )
                             {
                                 std::stringstream ss;
-                                for (auto& elem : message.data())
-                                    ss << std::setfill('0') << std::setw(2) << std::hex << static_cast<int>(elem) << " ";
-                                add_log(message.get_severity(), __FILE__, 0, ss.str());
+                                for( auto & elem : message.data() )
+                                    ss << std::setfill( '0' ) << std::setw( 2 ) << std::hex
+                                       << static_cast< int >( elem ) << " ";
+                                add_log( message.get_severity(), __FILE__, 0, ss.str() );
                             }
-                            if( ! enable_firmware_logs && fwlogger.get_number_of_fw_logs() == 0 )
-                            {
-                                fwlogger.stop_collecting();
+
+                            if( !enable_firmware_logs && fwlogger.get_number_of_fw_logs() == 0 )
```

Comment:
As I understand from the code, before if the user deactivate the log parsing, we continues dequeue until `get_number_of_fw_logs() == 0`
Now we will stop even we still have logs as our loop is `while( enable_firmware_logs )`
Is this change the required behavior?